### DISCUSSION TITLE
Support for Bar charts using time a time axis

### DISCUSSION THREAD

**Discussion by ranger-ross** [TanStack/react-charts#339] [Ideas]
First, let me mention that I am fairly new to this library (I am currently considering migrating to react-chart) so apologies in advance if my understanding is incorrect. 

Currently, I do not believe it is possible to have a "Bar" style chart using time series data.

I think this is a valid use-case, because there are often request to show data of a give "period".
For example, showing bars for each day of a month (screenshot example below)

![image](https://user-images.githubusercontent.com/25847436/215301502-4d05173e-6f29-4f7f-9a17-93843bf63f0c.png)

Is there an easy way to do this in react chart?
If not, maybe we can consider adding support for this :pray: ?
Thank you!

**Comment by GreenAsJade** [comment]
It seems possible - I'm in the midst of implementing one.  It's just primaryAxis scaleType "time" and secondaryAxis elementType bar.
